examples/pipelines/finetune_stable_diffusion/build_images.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+source ../docker_build.config
```

Comment:
The IMAGE_TAG should probably be a parameter of the script, right?

I'm more in favor of adding these config variables as parameters and just setting a default. Eg:
```
while [[ "$#" -gt 0 ]]; do case $1 in
  -n|--namespace) namespace="$2"; shift;;
  -r|--repo) repo="$2"; shift;;
  -t|--tag) tag="$2"; shift;;
  *) echo "Unknown parameter passed: $1"; exit 1;;
esac; shift; done

[ -n "${namespace-}" ] || namespace="ml6team"
[ -n "${repo-}" ] || repo="express"
[ -n "${tag-}" ] || tag="latest"
```

Reply:
Seems like a much neater solution than having a separate file for the default configs. Thanks for the suggestion